### ISSUE TITLE
Question about transfer learning

### ISSUE THREAD

**Issue by DongKeon** [keithito/tacotron#208]
Thank you for providing the open source :)

I would like to train with own dataset(English).

Is it possible to use transfer learning on the pre-trained model(LJ-dataset)?

I hope you can help me thank you.

**Comment by keithito** [comment]
Yes, that should be possible. Someone was able to train an Obama model with only 1.6 hours of data by fine-tuning the model trained on LJ Speech:
https://github.com/keithito/tacotron/issues/10#issuecomment-343015698
